owncloudApp/src/main/java/com/owncloud/android/ui/activity/LogHistoryActivity.kt

```diff
@@ -51,7 +50,7 @@ class LogHistoryActivity : ToolbarActivity() {
 
             fileLoggingTree()?.let {
                 logFragment = BothLogsFragment.newInstance(
-                    bothLogsTargetFileName,
+                    "${getString(R.string.app_name)}.log",
```

Comment:
I didn't debugged it, but is it that smart to use `R.string.app_name` ? What's about when the app has a name like "my Cloud" with space ?
Leave it like it was and all is fine

Reply:
I tried it and no problem at all with spaces in the log to share

[<img src="https://user-images.githubusercontent.com/15261363/43832186-df6aa322-9b06-11e8-99ce-f2d3d56a1542.png" width="350px"/>](https://user-images.githubusercontent.com/15261363/43832186-df6aa322-9b06-11e8-99ce-f2d3d56a1542.png)